core/vm.cpp

```diff
@@ -1704,6 +1704,10 @@ class Interpreter {
             case json::value_t::discarded: {
                 abort();
             }
+
+            case json::value_t::binary: {
```

Comment:
@timp87 Hello, This change here has some breaking changes on our CI/CD pipelines since few hours ago. I will paste the error snippet down below. Could you please fix this at your earliest convenience ? 

```core/vm.cpp: In member function 'void {anonymous}::Interpreter::otherJsonToHeap(const json&, bool&, {anonymous}::Value&)':
core/vm.cpp:1714:18: error: 'binary' is not a member of 'nlohmann::basic_json<>::value_t {aka nlohmann::detail::value_t}'
             case json::value_t::binary: {
                  ^
Makefile:159: recipe for target 'core/vm.o' failed
make: *** [core/vm.o] Error 1

Reply:
I reverted it, thanks for the heads up

Reply:
Thanks for taking care of it.

Reply:
I think the problem is the same as stated in my last comment.
Seems like you're bundled nlohmann-json is too old.